src/main/java/org/jenkinsci/plugins/buildtriggerbadge/IconFinder.java

```diff
@@ -79,6 +79,7 @@ private String internalFindForClass(Class clazz)
 		defineIconForCause("com.cloudbees.plugins.flow.FlowCause", "flow-cause.png");
 		defineIconForCause("com.cloudbees.jenkins.plugins.BitBucketPushCause", "bitbucket.png");
 		defineIconForCause("hudson.plugins.git.GitStatus$CommitHookCause", "git-hook-cause.png");
+		defineIconForCause("org.jenkinsci.plugins.urltrigger", "url-trigger-cause.png");
```

Comment:
https://github.com/jenkinsci/urltrigger-plugin/blob/master/src/main/java/org/jenkinsci/plugins/urltrigger/URLTriggerCause.java

Is the cause.

So 34aeab0 should fix this